DiceKeys/Views/Scanning/ScanDiceKey.swift

```diff
@@ -58,9 +59,12 @@ struct ScanDiceKey: View {
     }
 
     func onFrameProcessed(_ processedImageFrameSize: CGSize, _ facesRead: [FaceRead]?) {
-        self.frameCount += 1
-        self.processedImageFrameSize = processedImageFrameSize
-        self.facesRead = facesRead
+        DispatchQueue.main.async {
+            cameraFrameCountModel.frameCount += 1
+            facesReadOverlayModel.imageFrameSize = processedImageFrameSize
```

Comment:
Does this need an
```
  if facesReadOverlayModel.imageFrameSize != processedImageFrameSize {

  }
```

Or is SwiftUI smart enough not to trigger re-rendering if the frame doesn't change size?

Reply:
Don't think so, SwiftUI is smart enough. Need to verify by putting above condition.